regolith/helpers/u_milestonehelper.py

```diff
@@ -27,24 +27,27 @@ def subparser(subpi):
     if isinstance(subpi, GooeyParser):
         date_kwargs['widget'] = 'DateChooser'
 
-    subpi.add_argument("projectum_id", help="The id of the projectum.  If you just "
-                                            "specify this the program will return a "
-                                            "numbered list of milestones to select for "
-                                            "updating. #1 in the list is always used "
-                                            "to add a new milestone.")
-    subpi.add_argument("-c", "--current", action="store_true",
-                       help="only list current (unfinished, unpaused) milestones",
-                       )
-    subpi.add_argument("-i", "--index",
-                       help="Index of the item in the enumerated list to update. "
-                            "To update multiple milestones with the same edits "
-                            "(often used for finishing checklist items), "
-                            "please enter in the "
-                            "format of 2,5,7 or 3-7 for multiple indices.",
-                       type=str)
-    subpi.add_argument("-v", "--verbose", action="store_true",
-                       help="Increases the verbosity of the output.")
-    subpi.add_argument("-u", "--due-date",
+    subpi.add_argument("-u", "--milestone_uuid",
```

Comment:
let's use `-i` for this.  I use `-u` elsewhere for due date so it is good to keep it uniform.  Also, i for "id" seems to be more intuitive, and also we were using it for index before, so it all points towards that being a better idea.

Reply:
`-i` for the uuid is a great idea. changing.